frontend/public/components/secrets/create-secret.tsx

```diff
@@ -351,12 +351,14 @@ class ImageSecretFormWithTranslation extends React.Component<
       dataKey: getImageSecretKey(this.props.secretType),
       stringData: parsedData,
       authType: 'credentials',
+      disable: false,
     };
     this.onDataChanged = this.onDataChanged.bind(this);
     this.changeFormType = this.changeFormType.bind(this);
     this.onFormDisable = this.onFormDisable.bind(this);
   }
   onDataChanged(secretData) {
+    _.isError(secretData) ? null : this.onFormDisable(this.state.disable);
```

Comment:
I think the  `if` statement with negation or logical NOT (`!`) condition looks more readable for this case, and also since the first value of the ternary operator is not use anywhere.